<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabBrowser</class>
 <widget class="QWidget" name="tabBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
    background-color: rgb(211,211,211);
    color: rgb(88, 88, 90);
    selection-background-color: rgb(21, 165, 137);
    selection-color: rgb(245, 244, 244);
    font: 16pt
}

QLabel#error_label
{
    color: rgb(220, 130, 30);
}

QLineEdit, QTextEdit, QTableWidget
{
   background-color:  rgb(245, 244, 244)
}

QPushButton
{
	background-color: rgb(21, 165, 137);
    color: rgb(245, 244, 244);
}

QPushButton#home_button, QPushButton#parent_button, QPushButton#refresh_button
{
    background-color: rgb(245, 244, 244);
}

QTabWidget#info_tabs
{
     background-color: background-color: rgb(211,211,211);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_13">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>iRODS path: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_path">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>/zoneName/home/user</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="refresh_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/refresh.png</normaloff>../icons/refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="parent_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/arrow-up.png</normaloff>../icons/arrow-up.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="home_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../icons/home.png</normaloff>../icons/home.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="create_coll_button">
         <property name="text">
          <string>Create Collection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rename_button">
         <property name="text">
          <string>Rename/Move</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="upload_button">
         <property name="text">
          <string>Upload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="download_button">
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="delete_button">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="browser_table">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>250</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Status</string>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size [bytes]</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Checksum/Fingerprint</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Created</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modified</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_12">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTabWidget" name="info_tabs">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="metadata">
      <attribute name="title">
       <string>Metadata</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="meta_table">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Key</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Units</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="11" column="2">
          <widget class="QPushButton" name="add_meta_button">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="meta_units_field"/>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Units</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="meta_key_field"/>
         </item>
         <item row="13" column="2">
          <widget class="QPushButton" name="delete_meta_button">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="meta_value_field"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Key</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="QPushButton" name="update_meta_button">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Value</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="preview">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Preview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="preview_browser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="permissions">
      <attribute name="title">
       <string>Permissions</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="acl_table">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>User</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Zone</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Access</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Inherit</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>User name</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Zone</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Access</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Apply to all
in collection</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLineEdit" name="acl_user_field"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="acl_zone_field"/>
         </item>
         <item row="6" column="2">
          <widget class="QComboBox" name="acl_box">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>read</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>write</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>own</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>delete</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QComboBox" name="recursive_box">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>False</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>True</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QPushButton" name="add_acl_button">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Add/Update</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="owner">
           <property name="text">
            <string>Owner: </string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="owner_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="replicas">
      <attribute name="title">
       <string>Replicas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="replica_table">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Replica</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hierarchy</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Checksum</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size [bytes]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="no_meta_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="error_label">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
